collection-edifice.yaml

```diff
@@ -10,27 +10,27 @@ repositories:
   ign-fuel-tools:
     type: git
     url: https://github.com/ignitionrobotics/ign-fuel-tools
-    version: ign-fuel-tools5
+    version: ign-fuel-tools6
   ign-gazebo:
     type: git
     url: https://github.com/ignitionrobotics/ign-gazebo
-    version: main
+    version: ign-gazebo5
```

Comment:
Many of these branches don't exist, we should use `main` for the latest branches.